### DISCUSSION TITLE
Depend on typer-slim instead of typer

### DISCUSSION THREAD

**Discussion by BLKSerene** [explosion/spaCy#13538] [New Features & Project Ideas]
Hi, `typer` 0.12 has heavier dependencies, so is it possible for `spaCy` and `weasel` to depend on [`typer-slim`](https://github.com/tiangolo/typer?tab=readme-ov-file#dependencies) that has no dependencies instead?

**Comment by svlandeg** [comment]
Yes - that's a good point, we'll look into that!

**Comment by BLKSerene** [comment]
Solved by #13759.
